packages/server/src/db/index.js

```diff
@@ -138,7 +137,7 @@ async function getUser(id) {
         )
         .leftJoin('roles', 'roles.id', 'users.role_id')
         .leftJoin('agencies', 'agencies.id', 'users.agency_id')
-        .leftJoin('tenants', 'tenants.main_agency_id', 'agencies.main_agency_id')
+        .leftJoin('tenants', 'tenants.main_agency_id', 'agencies.id')
```

Comment:
Ensuring this join is as intended here. I believe this is the fk relationship but not 100% on the intent.

Reply:
It looks like this query is simply using this join to get information about the tenant associated with the user's agency. The agency associated with a user might not always be the main agency within that tenant, so I think this is supposed to be:
```suggestion
        .leftJoin('tenants', 'tenants.id', 'agencies.tenant_id')
```

Reply:
Actually, on second glance, there's a more direct path to the tenant associated with the user:
```suggestion
        .leftJoin('tenants', 'tenants.id', 'users.tenant_id')
```

Reply:
I think this one still needs to be updated- might have been resolved prematurely. 

Suggest joining the `tenants` table where `tenants.id = users.tenant_id` ([second suggestion](https://github.com/usdigitalresponse/usdr-gost/pull/1206/files#r1162110418) from the above comments).